src/oauth-backend.js

```diff
@@ -156,60 +171,63 @@ export default class OAuthBackend extends AuthBackend {
 				}
```

Comment:
I would also suggest replacing lines 169-170 with `this.deleteLocalUserInfo()` since it does exactly the same thing.

Unfortunately, I can't suggest this change because any commit didn't change these lines. Or can I?

Reply:
Try it?

Reply:
Here is what I do (I made a tiny screencast):

https://youtu.be/HfWYdqnU4fY

What am I missing?

Reply:
Ah, I see. If you want, you can edit the branch directly!

Reply:
Did it with pleasure! :)

Reply:
Feel free to merge when you think it's ready!